GitUI/GitExtensionsForm.cs

```diff
@@ -13,10 +14,9 @@ namespace GitUI
     /// <remarks>Includes support for font, hotkey, icon, translation, and position restore.</remarks>
     public class GitExtensionsForm : GitExtensionsFormBase
     {
-        private readonly WindowPositionManager _windowPositionManager = new WindowPositionManager();
-
+        private IWindowPositionManager _windowPositionManager = new WindowPositionManager();
+        private Func<IReadOnlyList<Rectangle>> _getScreensWorkingArea = () => Screen.AllScreens.Select(screen => screen.WorkingArea).ToArray();
```

Comment:
Why a func field and not just a method/property?

Also the name suggests singular, but really it's a list.

Reply:
Thank you, my internal StyleCop stuttered...

Reply:
I had a fresh look in a morning, and I don't think it would be correct to pluralise the name.
When I think about it, it is _one working area across multiple screens, that are represented by rectangles_. 
I agree there is a certain disconnect between the name and the return type, but some things don't lend themselves well for pluralisation. 
I'll rename it, if you feel that _working areas_ is notionally better.

As for _field vs property_ question - because this member is private use only, I feel that a field is sufficient. I would use a _property_ when it is part of a class' public surface, or a complex construct (though then the argument shifts into _property vs method_ territory).